temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` (Python Debugger) in committed code. It is generally used for debugging purposes during development and should not be included in production code.
```
Remove the line `import pdb`.
```